pages/Search/search.html

```diff
@@ -0,0 +1,86 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Ishita Keshawani" />
+    <meta name="description" content="Search page of Stunners Hub" />
+    <meta name="keywords" content="beauty store,e-commerce,react.js" />
+    <link rel="stylesheet" href="https://felizui.netlify.app/styles/index.css" />
+    <link rel="stylesheet" href="/pages/home/homepage.css" />
+    <link rel="stylesheet" href="/pages/products/products.css" />
+    <link rel="stylesheet" href="/pages/Search/search.css">
+    <script src="https://kit.fontawesome.com/7744bfb8db.js" crossorigin="anonymous"></script>
+    <link rel="icon" href="/assets/Stunners.png" />
+    <title>Search page</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <nav class="navbar semibold-font-weight">
+            <div class="openMenu"><i class="fa fa-bars"></i></div>
+            <a class="nav-icon-link nav-link link-no-style hide-icon" href="/index.html">
+                <img class="logoimage" src="/assets/Stunners.png" alt="" />
+                <span class="small-fontsize">Stunners Hub</span>
+            </a>
+            <a class="nav-link link-no-style nav-home" href="/index.html">Home</a>
+            <a class="nav-link link-no-style nav-products" href="/pages/products/products.html">Products</a>
+
+            <div class="mainMenu">
+                <div class="searchbar">
+                    <i class="fa-solid fa-magnifying-glass search-icon"></i>
+                    <input type="text" class="search-input" placeholder="Search on Stunners Hub" />
+                </div>
+                <a href="/pages/Authentication/login/login.html" class="link-no-style nav-link nav-icon-link">
+                    <i class="fas fa-user hide-icon"></i>
+                    <span class="small-fontsize">Login</span>
+                </a>
+                <a href="/pages/wishlist/wishlist.html" class="link-no-style nav-link nav-icon-link">
+                    <i class="fas fa-heart hide-icon"><span class="number-badge">0</span></i></i>
+                    <span class="small-fontsize">Wishlist</span>
+                </a>
+                <a href="/pages/cart/cart.html" class="link-no-style nav-link nav-icon-link">
+                    <i class="fas fa-cart-shopping hide-icon"><span class="number-badge">0</span></i></i>
+                    <span class="small-fontsize">Cart</span>
+                </a>
+
+                <div class="closeMenu">
+                    <i class="fa fa-times"></i>
+                </div>
+            </div>
+        </nav>
+        <div class="search-page">
+            <div class="search-result">Search results for "Liquid Lipstick" - 1 items</div>
+            <div class="cards-list">
+                <div class="card product-card">
+                    <div class="card-padding">
+                        <img class="product-img" src="/assets/product1.jpg" alt="" />
```

Comment:
Please provide alt description for accessibility

Reply:
done